lms/envs/common.py

```diff
@@ -1197,7 +1197,13 @@ def _make_mako_template_dirs(settings):
 # once the responsibility of XBlock creation is moved out of modulestore - cpennington
 XBLOCK_MIXINS = (LmsBlockMixin, InheritanceMixin, XModuleMixin, EditInfoMixin)
 
-# Allow any XBlock in the LMS
+# .. setting_name: XBLOCK_SELECT_FUNCTION
+# .. setting_default: prefer_xmodules
+# .. setting_description: Function to select XBlock when multiple matching entrypoints.
```

Comment:
Oops.  I think this line was meant to be deleted.

Reply:
:man_facepalming: oh man, very sorry about that, fixed now